src/networks/mainnet/mod.rs

```diff
@@ -79,9 +79,11 @@ pub static HEIGHT_INFOS: Lazy<HashMap<Height, HeightInfo>> = Lazy::new(|| {
         // Wed 20 Nov 23:00:00 UTC 2024
         make_height!(TukTuk, 4_461_240, get_bundle_cid("v15.0.0")),
         // Thu 10 Apr 23:00:00 UTC 2025
-        make_height!(Teep, 4_867_320, get_bundle_cid("v16.0.0")),
+        // TODO: Should we disable it until decided if its pushed or not?
```

Comment:
I think we should either unset or reset it

Reply:
The discussion is on-going in the implementer channel.